### PR TITLE
Add Buying Offsets, Helping Clients, Resources

### PR DESCRIPTION
**Pull request by chrismekim**
Figma: https://www.figma.com/file/bWWgCqrjStjy1ne99Vs6x9/Sanctuary-Sustainability?node-id=457%3A166

- Adds Buying Offsets, Helping Clients, and Resources.
- These are placeholder images while we wait for the actual images. 
- Doesn't include the nav (design is not finalized)

### CODE REVIEW

**Comment by brendonsanctuary** [comment]
New styling for Resources page.

Underline for section titles.
Black text for links.
Fade out hover to gray similar to footer. 

Screen Shot 2020-07-09 at 1 48 27 PM

**Comment by brendonsanctuary** [comment]
Desktop:
Lets make this graphic have a margin-top of 5rem instead of 10.

Screen Shot 2020-07-09 at 2 06 09 PM

**Comment by brendonsanctuary** [comment]
Pushing the Buying Offsets a little bit to the right.
Screen Shot 2020-07-09 at 3 00 26 PM

**Comment by brendonsanctuary** [comment]
Resources slight update.

Gray hover over similar to footer.
Screen Shot 2020-07-09 at 4 28 34 PM

**Comment by brendonsanctuary** [comment]
Mobile Margin for content on pages except Home and Resources

Screen Shot 2020-07-09 at 4 35 51 PM

**Comment by brendonsanctuary** [comment]
Something is going down here.

Screen Shot 2020-07-09 at 5 40 46 PM

**Comment by brendonsanctuary** [comment]
Tweak the placement of the word resources.

12 rem top margin before the links. 

Screen Shot 2020-07-09 at 5 43 45 PM

**Comment by brendonsanctuary** [comment]
12 rem margin-top before the links
Screen Shot 2020-07-09 at 5 46 13 PM

**Comment by brendonsanctuary** [comment]
Change the margin to 5 rem on this graphic. 

Screen Shot 2020-07-10 at 11 59 07 AM

**Comment by brendonsanctuary** [comment]
Mobile:
Footer on resources page should match the other pages.

Screen Shot 2020-07-10 at 12 23 13 PM

**Comment by brendonsanctuary** [comment]
Mobile:
Helping Clients Image size should be the same size as the Calculating Emissions. 
Screen Shot 2020-07-10 at 12 25 43 PM

Screen Shot 2020-07-10 at 12 25 36 PM

**Comment by brendonsanctuary** [comment]
Mobile:
Buying offsets is missing bottom margin. Should be the same as the other mobile pages exclude the homepage. 
Screen Shot 2020-07-10 at 12 26 40 PM

**Comment by brendonsanctuary** [comment]
Mobile:
Buying offsets graphic.
Margin top & bottom should be 2 rem. 
Screen Shot 2020-07-10 at 12 30 20 PM

**Comment by brendonsanctuary** [comment]
Desktop:
Move the Buying Offsets titles down. 

Screen Shot 2020-07-10 at 1 53 28 PM


Screen Shot 2020-07-10 at 1 53 21 PM

**Comment by brendonsanctuary** [comment]
Desktop Resource Margin:

Right now the margin is calculated from the top of the "Word Resources" Can we get it so 12 rem is calculated from the Bottom of the word "Resources"
Screen Shot 2020-07-10 at 1 56 17 PM

**Comment by brendonsanctuary** [comment]
Mobile:

Footer of resources is on the wrong grid. It should be the same as on every other page.

Screen Shot 2020-07-10 at 2 00 40 PM
Screen Shot 2020-07-10 at 2 00 38 PM

**Comment by brendonsanctuary** [comment]
Helping Clients Mobile:
Center the Image more. I think you have two competing div styles with right and left here. 

We can just make the top 30% and left 30%
Screen Shot 2020-07-10 at 2 03 12 PM
Screen Shot 2020-07-10 at 2 03 14 PM

File: src/clients.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        </p>
+      </section>
+     
+      <section class="content-container inner-content-padding-x  mxauto flex-col">
+        <p class="text-subtitle serif lg:col-8">
+          Studio Carbon Negative was initiated by Sanctuary Computer. Get in touch if you would like to know more information about Studio Carbon Negative.
+        </p>
+        <a href="https://www.sanctuary.computer/"><button class="mt2">&rarr; Visit Sanctuary Computer</button></a>
```
Comment (mokaymokay) [inline src/clients.html:119]: These buttons inside links are weird, I would use either button or link but not both.

File: src/clients.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+    <!-- Start Page Content -->
+    <header class="header mb5 vh100 md:mb6">
+      <div class="helping z1 col col-6 relative">
+        <h1 class="text-title serif">
+          Helping
+        </h1>
+      </div>
+      <div class="clients-header-image-container absolute col-5 md:col-2">
+        <img class="h100 w100 fit-cover" src="img/antarctica-B.jpg"/>
```
Comment (mokaymokay) [inline src/clients.html:26]: add an alt?

File: src/clients.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+              Most of the startups and early-stage businesses that we work with share a specific customer demographic: the early adopter. This an individual who is typically more prosperous, progressive, educated and well connected than other customers.
+            </p>
+            <p class="text-paragraph atlas">
+              In recent years, we've seeing a clear trend of Millennials and Gen Z consumers buying less at higher quality - the environmental effect of their purchase more and more often being one of their key decision making points.
+            </p>
+            <p class="text-paragraph atlas">
+              They're not easily greenwashed - they need to feel like the product can show real transparency and care in sustainability and environmental impact; but when done right, that consumer will become a loyal fan, spreading the brand to their community, who generally hold their opinion in high regard.
+            </p>
+            <div class="sustainability-diagram-container mt2 mb3 lg:mb0 lg:mt5">
+              <img class="h100 w100" src="img/sustainability-diagram.svg"/>
```
Comment (mokaymokay) [inline src/clients.html:59]: add an alt?

File: src/offsetting.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+          </div>
+        </div>
+      </section>
+     
+      <section class="content-container inner-content-padding-x  mxauto flex-col">
+        <p class="text-subtitle serif lg:col-8">
+          Studio Carbon Negative was initiated by Sanctuary Computer. Get in touch if you would like to know more information about Studio Carbon Negative.
+        </p>
+        <a href="https://www.sanctuary.computer/"><button class="mt2">&rarr; Visit Sanctuary Computer</button></a>
```
Comment (limsohee1002) [inline src/offsetting.html:120]: you should only use a tag here. maybe change button to span?

**Comment by brendonsanctuary** [comment]
This image should be a lil lower on the page

68747470733a2f2f63646e2e666561747572657065656b2e636f6d2f67656e657269632f623565343366313164353335643165612f5261636b4d756c74697061727432303230303731302d312d3131796563686e2e706e67

**Pull request merged**